src/modules/decklink/producer/decklink_producer.cpp

```diff
@@ -509,6 +509,12 @@ class decklink_producer : public IDeckLinkInputCallback
                 }
             }
 
+            if (new_timecode != core::frame_timecode::empty())
+            {
+                // Account for latency introduced by ffmpeg filter
+                new_timecode = new_timecode - (format_desc_.field_count > 1 ? 3 : 1);
```

Comment:
This is not going to be entirely accurate...

Reply:
Hm, no that's a bad idea, but I still think you can measure the filter latency using pts

Reply:
hmm... maybe this is good enough in practice

Reply:
leave it